daprdocs/content/en/developing-applications/building-blocks/state-management/query-state-store/query-cosmosdb-store.md

```diff
@@ -56,4 +67,7 @@ And to get a specific actor state such as "food", use the command:
 SELECT * FROM states WHERE states.id = 'mypets||cat||leroy||food'
 ```
 
-> **WARNING:** You should not manually update or delete states in the store. All writes and delete operations should be done via the Dapr runtime.
+{{% alert title="Warning" color="warning" %}}
+You should not manually update or delete states in the store. All writes and delete operations should be done via the Dapr runtime.
```

Comment:
```suggestion
You should not manually update or delete states in the store. All writes and delete operations should be done via the Dapr runtime. The one exception to this, is that it is often required to delete actor records in a state store, _once you know that these are no longer in use_, to prevent a build up of unused actor instance that may never be loaded again.
```